### DISCUSSION TITLE
맥북 m1용으로 빌드된 도커 이미지도 있나요?

### DISCUSSION THREAD

**Discussion by kimsw61** [naver/ngrinder#857] [Q&A]
안녕하세요 ngrinder를 aws / ec2에서 도커로 사용중입니다.

ec2 성능이 낮아 높은것을 쓰려다가 도커에서 돌아가니 맥북에서 돌리는게 나을거 같아서 m1 pro 맥북에서 동작을 시도하는데

WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

아래와 같은 문구가 뜹니다.

controller / agent 모두 마찬가지인 상태이며, 

controller의 경우 db transaction 처리에 뭔가 문제가 있는지 계속 지연되다가 에러가 발생했습니다.

liquibase: waiting for changelog lock.... 이 부분에서 계속 시도를 하다가 막혀서 exit가 뜨는 현상이 나타나서 이미지를 통째로 날리고 

다시 pull을 받아서 진행을 하고 있습니다만, 시간이 오래 걸리고 있어서 어떻게 될지 알 수 없습니다.

혹시 arm64 프로세스 빌드도 업로드가 가능할까요?

**Comment by donggyu04** [comment]
`liquibase: waiting for changelog lock.... `  에러는 이미지 문제가 아닌것 같습니다.
그냥 도커 이미지 말고 [최신버전](https://github.com/naver/ngrinder/releases/download/ngrinder-3.5.5-p1-20210531/ngrinder-controller-3.5.5-p1.war) 을 다운 받으시고 m1 노트북 환경에서 구동시켜 보시기 바랍니다.
저는 아래 사양에서 문제없이 구동해서 테스트 진행했습니다.

![image](https://user-images.githubusercontent.com/14273601/161730213-ce8f3ca2-bab1-49b0-8212-a9ccd6f247ea.png)

jdk 8 or 11 을 사용하시고, [war 파일 구동법](https://github.com/naver/ngrinder/discussions/819#discussioncomment-1212049) 코멘트 참조하셔서 구동해 보시기 바랍니다.
